tuareg.el

```diff
@@ -3097,30 +3086,52 @@ expansion at run-time, if the run-time version of Emacs does know this macro."
 (defvar show-paren-data-function); Silence the byte-compiler
 
 (defun tuareg--show-paren (orig-fun)
-  (if (or (tuareg--point-before-comment-p) (tuareg--point-after-comment-p))
-      nil
-    (funcall orig-fun)))
+  "Advice for `show-paren-data-function' to match comment delimiters."
+  (let ((here (point))
+        there)
+    (cond
+     ;; Immediately after end of a comment?
+     ((and (eq (char-before) ?\))
+           (eq (char-before (1- here)) ?*)
+           (save-excursion (forward-comment -1)
+                           (setq there (point))
+                           (forward-comment 1)
+                           (eq here (point))))
```

Comment:
See previous comment. (And where did those `setq`s come from? Not from my (otherwise flawed) patch...)